tests/examples/test1/dependencies.yaml

```diff
@@ -0,0 +1,69 @@
+files:
```

Comment:
Maybe we can rename `test1` to have a more descriptive name for what it is testing?

This seems to be a pretty large test file. If we could test the components separately with smaller dependency files, that might be nice. Perhaps this can be `test-integration` and you can add `test-conda-minimal` (a pure test of conda output with a trivial matrix), `test-requirements-minimal` (same for requirements), `test-matrix-minimal` (a minimal conda output with a nontrivial matrix), etc.